spec/lib/vanagon/project_spec.rb

```diff
@@ -165,6 +165,7 @@
       allow(Vanagon::Component::Source::Http).to receive(:valid_url?).with(http_yaml_uri).and_return(true)
       http_source = instance_double(Vanagon::Component::Source::Http)
       allow(Vanagon::Component::Source).to receive(:source).and_return(http_source)
+      allow(http_source).to receive(:verify)
```

Comment:
Do these need to explicitly return anything or no?

Reply:
if so, seems like it would be worth testing both good and bad responses.

Reply:
Ah yeah, if the values don't match, [it will just fail](https://github.com/puppetlabs/vanagon/blob/master/lib/vanagon/component/source/http.rb#L100) but it does return true if they do -- I'll update